packages/infrastructure-lib/src/acc_balances_bc/tigerbeetle_acc_bal_adapter.ts

```diff
@@ -266,6 +266,32 @@ export class TigerBeetleAccountsAndBalancesAdapter implements IAccountsBalancesA
         });
     }
 
+    async getJournalEntriesByTransferId(ledgerTransferId: string): Promise<AccountsAndBalancesJournalEntry[]> {
+        // Create request for TigerBeetle:
+        const accIdTB = this._uuidToBigint(ledgerTransferId);
+
+        // Invoke Client:
+        let transfers: TB.Transfer[] = [];
+        try {
+            const getAcc : AccountFilter = {
+                account_id: accIdTB,
+                timestamp_min: 0n,
+                timestamp_max: 0n,
+                limit: 100_000,//TODO need to make a config.
```

Comment:
Could we make this a config?

Reply:
I don't think we need getJournalEntriesByTransferId() anymore, if so maybe we can remove it alltogether?